### DISCUSSION TITLE
Add item to planning in progress

### DISCUSSION THREAD

**Discussion by byamagata** [microsoft/azure-boards-estimate#54] [Ideas]
Idea:
It would be great to be able to add a work item to a session in progress.  Especially for Offline poker sessions that span several days.

Use Case:
Scrum Master starts offline planning session on monday.  Tuesday, the PO realizes that one of the high priority items is not in the planning poker session, but people have already started voting.  Is easily able to add a new item to the session, and notify everyone that a new item has been added.

**Comment by dbarbary** [comment]
Yes, this is a great idea. also if a session could be started by entering a list of comma separated story numbers - in a similar way to adding a new session by selecting a query.
